### ISSUE TITLE
[Feature Request]: simpler example

### ISSUE THREAD

**Issue by jkitchin** [CederGroupHub/alabos#83]
### Feature Requested

It would be helpful to have an example alabos that people can use out of the box. I am trying to make one myself with https://claude-light.cheme.cmu.edu/api. This is a remote device with a REST API. It seems like it should be easy to make a Device that uses requests to make a measurement, and some Tasks for different kinds of measurements.  There isn't a concept of a sample for the device, you just set RGB levels, and measure intensities out.

### Proposed Solution

I am pretty close to a solution, and would be happy to make a pull request when it is working.

**Comment by idocx** [comment]
Agree! I will work on a simpler example this week. I plan to put it in the default project so that users can have a runnable code  after executing `alaos init`.

**Comment by idocx** [comment]
For the sample concept, alabos itself supports the execution of experiments without samples. But you may not be able to directly use the experiment submission script mentioned in the document: https://cedergrouphub.github.io/alabos/submit_experiments.html.

I can also add an example to the example folder to show how we can do that.

**Comment by jkitchin** [comment]
I have a working example alab that should work out of the box at https://github.com/jkitchin/alabos/tree/main/examples/claude-light.

It uses an instrument I created at http://claude-light.cheme.cmu.edu

I would be happy to get feedback on it. If you are interested, I could make a pull request.

**Comment by idocx** [comment]
> I have a working example alab that should work out of the box at https://github.com/jkitchin/alabos/tree/main/examples/claude-light.
> 
> It uses an instrument I created at http://claude-light.cheme.cmu.edu
> 
> I would be happy to get feedback on it. If you are interested, I could make a pull request.

Hi John! This is really amazing! I tried your code on my laptop. It worked perfectly. I checked the code. It is exactly how we expected users to program with AlabOS.

I am very interested in adding this example to the alabos example folder as it is. If you have time, could you create a pull request? Thank you very much!

**Comment by jkitchin** [comment]
I submitted a pull request at https://github.com/CederGroupHub/alabos/pull/86.

**Comment by sgbaird** [comment]
Just came across this and wanted to say, so great to see this!
